.goreleaser.yml

```diff
@@ -0,0 +1,59 @@
+project_name: kubectl-flyte
+before:
+  hooks:
+    - go mod download
+builds:
+  - env:
+      - CGO_ENABLED=0
+    main: ./cmd/kubectl-flyte/main.go
+    binary: kubectl-flyte
```

Comment:
@kumare3 Can you suggest a name for plugin name. https://krew.sigs.k8s.io/docs/developer-guide/develop/naming-guide/

Reply:
I think this is fine... we do want the usage to be `kubectl flyte get -n my-namespace` to retrieve flyte workflows